ci-operator/jobs/infra-periodics.yaml

```diff
@@ -308,6 +308,35 @@ periodics:
     - name: token
       secret:
         secretName: github-credentials-openshift-bot
+- agent: kubernetes
+  labels:
+    ci.openshift.io/role: infra
+  extra_refs:
+  - base_ref: master
+    org: openshift
+    repo: release
+  cron: "30 10 * * 2"  # Run at 10:30 UTC each Tuesday
+  decorate: true
+  name: periodic-prow-auto-config-brancher
+  spec:
+    containers:
+    - command: ["/bin/sh","-c"]
+      ## To be updated
+      args: ["pwd && ls"]
```

Comment:
@stevekuznetsov Has it to be inline script?

I would like to solve the $HOME (pod ENV) issue in container and use `.netrc` (will be part of the script) there to avoid secret expose with something like:

`sed -e "s/token/$(cat /etc/github/token)/" /home/.netrc`

We are using bash now ... and lose `CENSOR`.

Reply:
Solution to inline: https://github.com/docker-library/redmine/issues/52

Reply:
Injected is fine!

Reply:
If the EnsurePR means you need Go, then we can use Go -- I just wanted us to try the easiest thing first

Reply:
https://github.com/openshift/ci-tools/pull/167